myscript.js

```diff
@@ -1,29 +1,30 @@
 //list item template
-var itemTemplate = 
+var ITEM_TEMPLATE =
     '<a href="#" class="list-group-item"> \
         <input type="checkbox" onclick="completeItem(this)" value=""> \
         <span class="badge">A</span> \
-        <button class="delete-button" onclick="deleteItem(this)">Delete</button></a>';
+        <button class="delete-button" onclick="deleteItem(this)">Delete</button> \
+        <span class="item-text"></span></a>';
 
 //addItem function
-var addItem = function () {
-    var list=$("#notCompletedItems");
-    var newItem=$(itemTemplate); 
-    var userInput = document.getElementById("text-input").value;
-    //put text input into the innerHTML of the newItem
-    newItem.append(userInput);
-    list.append(newItem);
-    //TODO clear the text input box
-    document.getElementById("text-input").value = '';
+function addItem() {
+    var $list = $('#notCompletedItems');
+    var userInput = $('#text-input').val();
+
+    var $newItem = $(ITEM_TEMPLATE);
+
+    $newItem.find('.item-text').text(userInput);
+    $list.append($newItem);
+    $('#text-input').val('');
 };
 
-var deleteItem = function(element){
+function deleteItem(element) {
     $(element).parent().remove();
 }
 
-var completeItem = function(checked) {
-    var doneItem = $(checked).parent();
-    var done =$("#completedItems");
-    done.append(doneItem);
+function completeItem(checked) {
```

Comment:
Same stuff, and made the variables `$doneList` so it's clear it's a jQuery object